static/scss/partials/onboarding.scss

```
@@ -393,29 +393,57 @@ firefox-private-relay-addon[data-addon-installed="true"] + firefox-private-relay
 }
 
 .c-premium-onboarding-action-completed {
-    padding: $spacing-md;
+    padding: $spacing-lg;
     padding-inline-end: $spacing-lg;
     background-color: $color-white;
     display: none;
     align-items: center;
     border-radius: $border-radius-md;
-    width: auto;
-
-    samp {
-        margin-bottom: 0;
-        color: $color-informational;
-        max-width: 100%;
-        word-break: break-all;
+    width: 100%;
 
-        @include text-title-3xs;
+    @media #{$mq-md} {
+        width: auto;
     }
 
     &.is-visible {
         display: inline-flex;
     }
 
-    img {
-        margin-right: $spacing-sm;
+    p {
+        margin: 0;
+    }
+
+    .c-premium-onboarding-subdomain-set-module {
+        font-family: $font-stack-firefox;
+        text-align: center;
+        font-weight: 600;
+        margin: 0 auto;
+    }
+
+    .c-premium-onboarding-label {
+        color: $color-dark-gray-05;
+    }
+
+    .c-premium-onboarding-label::before {
+        width: $layout-2xs;
+        height: $layout-2xs;
+        display: inline-block;
+        content: "";
+        background-image: url("/static/images/icon-check.svg");
+        background-repeat: no-repeat;
+        background-position: center center;
+        padding-right: $spacing-sm;
+    }
+
+    .c-premium-onboarding-subdomain {
+        @include text-body-lg;
+        font-weight: bold;
```

nitpick: Font-weights should be numeric. 
```suggestion
        font-weight: 700;
```

---

Also, egg on my face - I checked the code base for any other outliers. Could you resolve these while your making your updates?

- https://github.com/mozilla/fx-private-relay/blob/main/static/scss/partials/onboarding.scss#L51
- https://github.com/mozilla/fx-private-relay/blob/main/static/scss/partials/onboarding.scss#L98

---

Oh, I didn't know that we did that! Should we see if we can update [Protocol's CSS coding guide](https://protocol.mozilla.org/docs/css-guide.html#units)? Or is that just because that's how we load our fonts?

---

> Oh, I didn't know that we did that! Should we see if we can update [Protocol's CSS coding guide](https://protocol.mozilla.org/docs/css-guide.html#units)? Or is that just because that's how we load our fonts?

We don't but we should. I'll update the README.